gapir/cc/function_table.h

```diff
@@ -49,23 +55,23 @@ class FunctionTable {
   inline Function* lookup(Id id);
 
  private:
-  // Map of the supported function ids to the actual function implementations
-  std::unordered_map<Id, Function> mFunctions;
+  // Array of the actual function implementations by function ID
+  Function mFunctions[65536];
```

Comment:
Function mFunctions[65536] = {};
Then you shouldnt have to zero-initailize in the constructor by hand.

Reply:
Good idea, I forgot we're using c++11